### DISCUSSION TITLE
Telegram Notifications

### DISCUSSION THREAD

**Discussion by iHumberto** [seerr-team/seerr#358] [Q&A] [answered]
Hi guys,

I've set the telegram notifications, but there is an option called Bot Username (Allow users to also start a chat with your bot and configure their own notifications) but I don't know how to set this and can't find accross the web how to do it.

![image](https://user-images.githubusercontent.com/26930588/227745669-d0109f39-ffa7-45a4-b800-d8f5271d1565.png)

**Comment by fallenbagel** [comment, marked as answer]
Bot username should be your telegram bot's username

**Comment by iHumberto** [comment]
Oh... Thanks.... I thought the user could set the options in the bot, but he has to do that in the jellyseerr settings. Thanks for the help.

**Reply by fallenbagel** [reply]
User can only set chatid, and what type of notifications they want in their own profile settings
